apps/ensrainbow/src/ingest.ts

```diff
@@ -73,8 +78,6 @@ async function loadEnsNamesToLevelDB(): Promise<void> {
           }
         }
       }
-    } else {
-      bar.tick();
     }
```

Comment:
Now, that we can the `else` clause gone, we could flatten these nested ifs a bit. What do you think, @djstrong?

From:

https://github.com/namehash/ensnode/blob/1f63b6f2694bfa488efe2a93677acfae9d5962d5/apps/ensrainbow/src/ingest.ts#L60-L82

to:

```ts
if (!isCopySection) {
  continue;
}

const parts = line.trim().split("\t");
if (parts.length !== 2) {
  continue;
}

const [hashVal, name] = parts;

if (!isHex(hashVal) || !name) {
  continue;
}

try {
  const hashBytes = Buffer.from(hexToBytes(hashVal));
  batch.put(hashBytes, name);
  batchSize++;

  if (batchSize >= MAX_BATCH_SIZE) {
    await batch.write();
    batch = db.batch();
    batchSize = 0;
  }
  bar.tick();
} catch (e) {
  console.error(`Error processing hash: ${e} '${hashVal}'`);
}
```